packages/app/src/device/constants/response-status.constant.ts

```diff
@@ -10,10 +10,11 @@ export enum ResponseStatus {
   NoContent = 204,
   BadRequest = 400,
   NotFound = 404,
-  PhoneLocked = 403,
+  PhoneLocked = 403, //Forbidden
```

Comment:
is the comment necessary :)?

Reply:
nope :D but we don't use regular meaning of this status :D